adafruit_macropad.py

```diff
@@ -397,9 +397,8 @@ def keys(self) -> Keys:
         * ``key_number``: the number of the key that changed. Keys are numbered starting at 0.
         * ``pressed``: ``True`` if the event is a transition from released to pressed.
         * ``released``: ``True`` if the event is a transition from pressed to released.
-                        ``released`` is always the opposite of ``pressed``; it's provided
-                        for convenience and clarity, in case you want to test for
-                        key-release events explicitly.
+            ``released`` is always the opposite of ``pressed``; it's provided for convenience
+            and clarity, in case you want to test for key-release events explicitly.
```

Comment:
Normally 4 spaces from reference is _exactly_ what sphinx wants for things like multiline `:param:` descriptions, but it seems the for unordered lists, the text must start at the same place:

```suggestion
        * ``released``: ``True`` if the event is a transition from pressed to released.
          ``released`` is always the opposite of ``pressed``; it's provided for convenience
          and clarity, in case you want to test for key-release events explicitly.
```

Reply:
If I understand correctly, it should look like this for the unordered list:

```
        * ``released``: ``True`` if the event is a transition from pressed to released.
          ``released`` is always the opposite of ``pressed``; it's provided for convenience
          and clarity, in case you want to test for key-release events explicitly.
```

But when I build it puts it on the same line, so I'm unsure what to do next.  Thanks for the review and any suggestions.

![Screen Shot 2022-09-15 at 4 49 48 AM](https://user-images.githubusercontent.com/67276/190373417-9cea20dd-2c0d-459e-ad8f-22a9af863547.png)

Reply:
Oh, do you want it to be on a new line entirely, like:

```text
Function description here
Function description here
Function description here

    * Bullet point text here
    * Bullet point text here
    
Explanation of bullet point text

Reply:
Otherwise, I think having it on the same line is the way that sphinx just naturally formats multiline things like this.  If you think the above option is better, that should be doable.

Reply:
Ok, that helps - having the explanation of the bulleted text not indented makes a lot more sense.

Thanks, I'll get this fixed.